spec/requests/bragi/items_api_spec.rb

```diff
@@ -150,6 +150,36 @@ module Bragi
           expect(json['data'][5]['id']).to eq item3.id.to_s
           expect(json['data'][6]['id']).to eq item2.id.to_s
         end
+
+        it 'fetches by after_id' do
+          get '/items', params: { filter: { after_id: item6.id } }, headers: { 'Authorization' => 'authorized_user' }
+
+          expect(response).to have_http_status(200)
+          json = JSON.parse response.body
+          expect(json['data'].size).to eq 3
+          json['data'].each do |item|
+            expect(item['attributes']['status']).to be_in(%w[unplayed playing])
```

Comment:
It'd be good to check that the items you're expecting to be in there are the actual ones that were returned.